feed_parser/src/main/java/Maalgm/FeedParser.java

```diff
@@ -0,0 +1,59 @@
+package Maalgm;
```

Comment:
package name are all lowercase according to Google Java Style.

https://google.github.io/styleguide/javaguide.html